### PR TITLE
FixCommand - fix wrong message for dry-run

### PR DESCRIPTION
**Pull request by SpacePossum**
For RFC https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/967

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Better to go to 1.11 IMO.

**Comment by keradus** [comment]
1.10 is fine, is this comment makes wrong impression to the users.

:+1:

**Comment by GrahamCampbell** [comment]
:+1: I guess.

**Comment by Soullivaneuh** [comment]
Nice little catch @SpacePossum. :+1:

File: Symfony/CS/Console/Command/FixCommand.php [hunk 1]
```diff
@@ -422,7 +422,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 }
 
                 $fixEvent = $this->stopwatch->getEvent('fixFiles');
-                $output->writeln(sprintf('Fixed all files in %.3f seconds, %.3f MB memory used', $fixEvent->getDuration() / 1000, $fixEvent->getMemory() / 1024 / 1024));
+                $output->writeln(sprintf('%s all files in %.3f seconds, %.3f MB memory used', $input->getOption('dry-run') ? 'Checked' : 'Fixed', $fixEvent->getDuration() / 1000, $fixEvent->getMemory() / 1024 / 1024));
```
Comment (Soullivaneuh) [inline Symfony/CS/Console/Command/FixCommand.php]: Maybe this call could be multi-lines for better readability

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**